include/libdnf5/rpm/package_query.hpp

```diff
@@ -618,13 +618,27 @@ class PackageQuery : public PackageSet {
     // @replaces libdnf/sack/query.hpp:method:addFilter(int keyname, int cmp_type, int match) - cmp_type = HY_PKG_LATEST_PER_ARCH
     void filter_latest_evr(int limit = 1);
 
+    /// Group packages by `name`. Then within each group, keep packages that correspond with up to `limit` of (all but) latest `evr`s in the group.
+    ///
+    /// @param limit            If `limit` > 0, keep `limit` number `evr`s in each group.
+    ///                         If `limit` < 0, keep all **but** `limit` last `evr`s in each group.
+    /// @since 5.2
+    void filter_latest_evr_ignore_arch(int limit = 1);
```

Comment:
I am thinking about naming. 
Important information is `Group packages by `name` or `Group packages by `name` and `arch`. But how it fits with
`ignore_arch`?
`latest` and `latest_evr` -  I think latest is always according `evr`.

Reply:
Yeah, I had `filter_latest_any_arch` in an earlier draft, do you think that's better than `filter_latest_ignore_arch`? I agree that `ignore_arch` is not ideal since `ignorearch` has an established meaning in our project.

I don't like simply `filter_latest` as much since `filter_latest_unresolved_advisories` also groups packages by arch; it seems that grouping by name+arch is common for the two existing `filter_latest_*` functions.

Reply:
`filter_earliest_evr_any_arch` sounds good to me. Especially when looking to the current state of sources in the `package_query.hpp` where the docstrings for `filter_latest_evr` are followed by the `filter_earliest_evr_any_arch` that gives a clear and coherent feeling.